rootfs/strap.py

```
@@ -74,6 +79,10 @@
         if parts[2] == "recompress":
             recompress = True
 
+if not re.match(r'^[a-zA-Z0-9._+\-]+$', package):
+    print("  ==> Error: invalid package name")
+    exit(1)
```

Ruff (0.15.17)

[warning] Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)